src/handlers/wallet/exchanges/test_exchange.rs

```diff
@@ -0,0 +1,106 @@
+use {
+    crate::handlers::wallet::exchanges::{
+        BuyTransactionStatus, ExchangeError, ExchangeProvider, GetBuyStatusParams,
+        GetBuyStatusResponse, GetBuyUrlParams,
+    },
+    crate::state::AppState,
+    crate::utils::crypto::Caip19Asset,
+    axum::extract::State,
+    once_cell::sync::Lazy,
+    serde::{Deserialize, Serialize},
+    std::sync::Arc,
+};
+
+pub struct TestExchange;
+
+const TEST_EXCHANGE_URL: &str = "https://appkit-pay-test-exchange.reown.com";
+
+static CAIP_19_SUPPORTED_ASSETS: Lazy<Vec<Caip19Asset>> = Lazy::new(|| {
+    vec![
+        Caip19Asset::parse("eip155:11155111/slip44:60").unwrap(),
```

Comment:
`Expect` would be better here to catch errors in case of a panic.
```suggestion
Caip19Asset::parse("eip155:11155111/slip44:60").expect("Failed to parse CAIP-19 assets list"),
```